lib/Infer/ExhaustiveSynthesis.cpp

```diff
@@ -119,9 +175,9 @@ std::vector<Inst *> matchWidth(Inst *I, unsigned NewW, InstContext &IC) {
   return { I };
 }
 
-void addGuess(Inst *RHS, int MaxCost, std::vector<Inst *> &Guesses,
-              int &TooExpensive) {
-  if (souper::cost(RHS) < MaxCost)
+void addGuess(Inst *RHS, float MaxCost, std::vector<Inst *> &Guesses,
+              float &TooExpensive, CostFunctionType &Cost) {
+  if (Cost(RHS, false) < MaxCost)
```

Comment:
when passing a boolean constant, follow the convention of including the name of the flag as a comment